buildSrc/src/main/kotlin/dev/nolij/zumegradle/JarCompressing.kt

```diff
@@ -143,22 +121,18 @@ private fun processClassFile(bytes: ByteArray, classFileSettings: ClassShrinking
 		}
 	}
 	
-	val isProGuardAnnotation = { annotationNode: AnnotationNode -> 
-		annotationNode.desc.startsWith("Ldev/nolij/zumegradle/proguard/")
-	}
-	
-	classNode.invisibleAnnotations?.removeIf(isProGuardAnnotation)
-	classNode.fields.forEach { fieldNode ->
-		fieldNode.invisibleAnnotations?.removeIf(isProGuardAnnotation)
-	}
-	classNode.methods.forEach { fieldNode ->
-		fieldNode.invisibleAnnotations?.removeIf(isProGuardAnnotation)
-	}
-
 	if (classNode.invisibleAnnotations?.any { it.desc == "Lorg/spongepowered/asm/mixin/Mixin;" } == true) {
 		classNode.methods.removeAll { it.name == "<init>" && it.instructions.size() <= 3 } // ALOAD, super(), RETURN
 	}
 
+	classNode.invisibleAnnotations?.removeIf { it.desc != "Lorg/spongepowered/asm/mixin/Mixin;" }
```

Comment:
Why

Reply:
invisible annotations are pretty much unused except for `@Mixin` which they made invisible for some reason (while the rest of the injector annotations are visible for some reason)

Reply:
Still don't like it. Should check against a list of annotations to strip, not annotations to keep.